plugin/status.go

```diff
@@ -0,0 +1,19 @@
+package plugin
+
+type Status struct {
+	DriverName     string
+	MeshDriverName string `json:"MeshDriverName,omitempty"`
+	Version        int
+}
+
+func NewStatus(address, meshAddress string, isPluginV2 bool) *Status {
+	status := &Status{
+		DriverName:     pluginNameFromAddress(address),
+		MeshDriverName: pluginNameFromAddress(meshAddress),
+		Version:        1,
```

Comment:
Minor/Nitpick: replace `1` with a constant, esp. as used in `prog/weaver/http.go` in `{{if eq .Plugin.Version 1}}`